arangod/RestHandler/RestBatchHandler.h

```diff
@@ -81,6 +82,15 @@ class RestBatchHandler : public RestVocbaseBaseHandler {
 
   // extract the next part from a multipart message
   bool extractPart(SearchHelper*);
+
+private:
+  bool executeNextHandler();
+  void processSubHandlerResult(std::shared_ptr<RestHandler> handler);
+
+  MultipartMessage _multipartMessage;
+  SearchHelper _helper;
+  size_t _errors;
```

Comment:
`_errors` should be initialized in the initializer list of RestBatchHandler, but not in `executeHttp()`